ocr_service/tests/api/test_ocr_endpoints.py

```
@@ -0,0 +1,17 @@
+import pytest
+from fastapi.testclient import TestClient
+from main import app
+
+client = TestClient(app)
+
+def test_root_endpoint():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_root_endpoint`

Add return type annotation: `None`

(ANN201)